src/checkers/jetbrainschecker.py

```diff
@@ -33,7 +34,12 @@ async def check(self, external_data: ExternalBase):
             result = await response.json()
             data = result[code][0]
 
-        release = data["downloads"]["linux"]
+        arch = (
+            external_data.checker_data["arch"]
```

Comment:
`ExternalData` objects already have `arches` property on them, you can access it as `external_data.arches[0]`, and it also defaults to `x86_64`. No need to re-introduce similar options just for this specific checker.

Reply:
Good to know. I'll drop the property and wire it up as you suggested.

Reply:
Just to confirm, would external_data.arches[0] return the architecture of the currently built package variant? I'm unclear why this would have to be an array.

Reply:
It refers to `only-arches` property of the source, which is an array of arches the source is used on, although the array usually contains only one item.